src/osd/PGLog.h

```diff
@@ -146,10 +147,10 @@ struct PGLog : DoutPrefixProvider {
    * plus some methods to manipulate it all.
    */
   struct IndexedLog : public pg_log_t {
-    mutable ceph::unordered_map<hobject_t,pg_log_entry_t*> objects;  // ptrs into log.  be careful!
-    mutable ceph::unordered_map<osd_reqid_t,pg_log_entry_t*> caller_ops;
-    mutable ceph::unordered_multimap<osd_reqid_t,pg_log_entry_t*> extra_caller_ops;
-    mutable ceph::unordered_map<osd_reqid_t,pg_log_dup_t*> dup_index;
+    mutable std::unordered_map<hobject_t,pg_log_entry_t*> objects;  // ptrs into log.  be careful!
```

Comment:
here, and in many other locations: missing a blank after the comma
(copied from the pre-fix code)

Reply:
I intentionally did not change the coding style because I thought it's out of this PR's scope.

Reply:
As you are touching these same lines, I'd vote for fixing this.

Reply:
OK, I'll wait for Jenkins to finish (to avoid interrupting it), and then I'll amend to add the spaces.

Reply:
Note that there are a few other instances where a space could be added.